dev/com.ibm.ws.org.apache.cxf.cxf.rt.databinding.jaxb.3.2/src/org/apache/cxf/jaxb/JAXBDataBinding.java

```diff
@@ -392,29 +411,54 @@ public synchronized void initialize(Service service) {
 
             if (col.getXmlSchemas().length > 1) {
                 // someone has already filled in the types
+                if (LOG.isLoggable(Level.FINE)) { // Liberty change begin
```

Comment:
Maybe continue using isLoggableFine instead of checking again?

Reply:
yes, will check for boolean here